EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTCollections.cc

```diff
@@ -2,14 +2,21 @@
 
 #include "GMTCollections.h"
 
+#include <sstream>
+
 namespace l1t {
    namespace stage2 {
       GMTCollections::~GMTCollections()
       {
          event_.put(std::move(regionalMuonCandsBMTF_), "BMTF");
          event_.put(std::move(regionalMuonCandsOMTF_), "OMTF");
          event_.put(std::move(regionalMuonCandsEMTF_), "EMTF");
-         event_.put(std::move(muons_), "Muon");
+         event_.put(std::move(muons_[0]), "Muon");
+         for (int i=1; i<6; ++i) {
+            std::stringstream ss;
+            ss << "MuonCopy" << i;
```

Comment:
just "MuonCopy"+std::to_string(i) 

should be sufficient (and much faster) here)